src/LazyOperations/SymmetricIntervalHull.jl

```diff
@@ -237,3 +237,14 @@ function get_radius!(sih::SymmetricIntervalHull{N},
     end
     return sih.cache[i]
 end
+
+# Faster support function calculation for sev and SymmetricIntervalHull
+function ρ(d::SingleEntryVector{N}, H::SymmetricIntervalHull{N}) where {N<:Real}
+    @assert length(d) == dim(H) "a $(d.n)-dimensional vector is " *
+                                "incompatible with a $(dim(H))-dimensional set"
+    if d.v < zero(N)
+        return d.v * (- radius_hyperrectangle(H, d.i))
+    else
+        return d.v * (+ radius_hyperrectangle(H, d.i))
```

Comment:
What about my second comment?

Reply:
Yes, i agree that the `abs` can be used instead of the branch. And it may be even slightly more efficient. I'll compare them.

Reply:
Actually there is a difference: if `d.v` is zero, `abs(d.v)` returns zero, but the branch adds `d.v * (+ radius_hyperrectangle(H, d.i))`.

Reply:
I tried them, and there is no difference in speed. But the code is a shorter with the `abs` so i decided to keep it.

Reply:
By the same token we can use `abs` in the general case, i mean in [this function](https://github.com/JuliaReach/LazySets.jl/blob/master/src/Interfaces/AbstractHyperrectangle.jl#L289) lines 295-299.

Reply:
I prefer the old version because the `abs` version now uses two multiplications instead of one. I was probably thinking about the support vector here. We can keep it here now that it is done, but I would not change it in the other method.